app/[lang]/chapters/page.tsx

```diff
@@ -22,13 +22,13 @@ export default async function ChaptersPage({ params }) {
             Chapters
           </h1>
 
-          <p className="pb-8 pl-10 pr-10 pt-4 text-center text-2xl sm:text-3xl lg:text-4xl">
+          <p className="pb-2.5 pl-10 pr-10 pt-4 text-center text-2xl sm:text-3xl lg:text-4xl">
             Explore the mysteries of Satoshi and learn about Bitcoin along the
             way.
```

Comment:
Lets go ahead and bite the bullet on making these translation keys

Reply:
Sure, I will add it. What about the newly added text?
I changed the previous 'demo_Disclaimer' translation keys to only 'Disclaimer' for English but the other languages still have the old keys so do I update the keys or we should only update the keys when the translation within them is being updated?
I think we should formulate a plan for updating the translations.

Reply:
I tried to add the the title and chapter page description as translation keys but because this component is not a client component it is not possible to use the 'useLang' hook. One work around could be adding the text as component